### PR TITLE
[#655] Completion of README.md Translation

### PR DESCRIPTION
**Pull request by ChobobDev**
I have managed to translate README.md 

Please leave comment if there are any typos or misleading information of the translation.

Thank you

### CODE REVIEW

File: i18n/README-EN.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ CB-Tumblebug welcomes improvements from all contributors, new and experienced!
 
 ### (1) Source Code based Installation and Execution
 
-- 개요
+- Table of Contents
   - Inatall tools and packages required
```
Comment (seokho-son) [inline i18n/README-EN.md:113]: I found some Typos 
Inatall -> Install or Installation

**Review by seokho-son — COMMENTED** [review]
@ChobobDev Thank you for the huge efforts to globalization of this project !!
In general, LGTM.  
I found some typos, please fix typos :)

Environmentla - Environmental
cnofiguring - configuring
Trubleshooting - Troubleshooting
reslove - resolve

I think we need to sync English readme with the Korean readme. 
I will invite contributors !

Again thank you very much ! 👍

File: i18n/README-EN.md (hunk 1 above)
Comment (ChobobDev) [reply i18n/README-EN.md:113]: Thank you for the reviews,

I did not realize there were so many typos 

I Will work on it ASAP

Thank you Again

**Comment by ChobobDev** [comment]
I have used VS Code Extensions so I believe there are no Typos on this document.

Again, Thank you for your hard work in finding my typos on this markdown file.

Thank you

**Pull request merged**